src/main/kotlin/org/partiql/scribe/ExcludeBindings.kt

```diff
@@ -0,0 +1,128 @@
+package org.partiql.scribe
```

Comment:
(self-review) Functions in this file were copied from `RedshiftRewriter` and moved the outer package namespace since we're reusing this logic for every target.